### PR TITLE
Release RSS feed

### PR DESCRIPTION
**Pull request by kevgo**
@charlierudolph @allewun @ricmatsui 

Adding a link to our release RSS feed so that people never miss a new release.

### CODE REVIEW

**Comment by allewun** [comment]
Would this be simpler/cleaner as the distinctive RSS badge?

**Comment by kevgo** [comment]
Good idea. I have created #635 to track this separately. I don't want to butcher @redouglas' designs here without a good reason to do so  :)

File: website/install.md [hunk 1]
```diff
@@ -72,3 +72,11 @@ Git Town runs anywhere Git and [Bash](https://www.gnu.org/software/bash/bash.htm
 #### Install autocompletion
 
 * for [Fish shell](http://fishshell.com): `git town install-fish-autocompletion`
+
+
+#### Notifications about new releases
+
+* Subscribe to our
+  [release RSS feed](https://github.com/Originate/git-town/releases.atom)
```
Comment (allewun) [inline website/install.md]: "RSS release feed" reads better to me (and cool! I didn't know GitHub did that)

File: website/install.md [hunk 2]
```diff
@@ -72,3 +72,11 @@ Git Town runs anywhere Git and [Bash](https://www.gnu.org/software/bash/bash.htm
...
+  to never miss a new release!
+  If you prefer email notifications, please try https://sibbell.com.
```
Comment (allewun) [inline website/install.md]: Linkify? `... please try Sibbell.`

**Comment by allewun** [comment]
LGTM

File: website/install.md (hunk 2 above)
Comment (kevgo) [inline website/install.md]: Done

File: website/install.md (hunk 1 above)
Comment (kevgo) [inline website/install.md]: I think the current way reads better. @charlierudolph want to chime in with a third (and deciding) opinion here?

File: website/install.md [hunk 3]
```diff
@@ -72,3 +72,11 @@ Git Town runs anywhere Git and [Bash](https://www.gnu.org/software/bash/bash.htm
 #### Install autocompletion
 
 * for [Fish shell](http://fishshell.com): `git town install-fish-autocompletion`
+
+
+#### Notifications about new releases
+
+* Subscribe to our
+  <a href="https://github.com/Originate/git-town/releases.atom)
```
Comment (allewun) [inline website/install.md]: Syntax error (hybrid HTML+Markdown)

File: website/install.md (hunk 1 above)
Comment (charlierudolph) [inline website/install.md]: how about just "release feed"? We are giving a link to an `.atom` and Atom is an RSS alternative.

File: website/install.md (hunk 3 above)
Comment (charlierudolph) [inline website/install.md]: Good catch

Comment (charlierudolph) [inline website/install.md]: The close paren needs to be removed and the part of the sentence to "never miss a new release!" needs to be moved to after the link

**Comment by redouglas** [comment]
@charlierudolph @allewun updated the RSS copy.

**Comment by charlierudolph** [comment]
Thanks Robert. Only thing left is resolving this [comment](#diff-for-comment-39007760)

**Comment by kevgo** [comment]
Addressed the last comment

**Comment by charlierudolph** [comment]
Looks good to me

**Pull request closed without merging**